omegaconf/_utils.py

```diff
@@ -465,7 +465,7 @@ def get_dict_key_value_types(ref_type: Any) -> Tuple[Any, Any]:
 
     key_type: Any
     element_type: Any
-    if ref_type is None:
+    if ref_type is None or ref_type == Dict:
```

Comment:
so we make the None here, and Union[str, Enum] and Any later.
seems like this can be simplified more returning (Union[str, Enum], Any) here in the first place.
(same for list).

Reply:
In order to simplify this as returning Union[str, Enum] and Any I have to change more logic in dictconfig and listconfig. I think for now it should be better this way.

Reply:
Did you push this?
I just spent a few minutes trying to figure that out and I don't think you did.

Reply:
No I didn't. I changed the test as you requested above and I was looking at changing this.

Reply:
Gotcha. send the update and we can discuss over the changes.

Reply:
@omry I don't know if returning Union[str, Enum] and Any would simplify this now. It would mean changing more internal logic like changing ref_type to Any as default.

Reply:
besides the fact that it means changing code, isn't Any a better default than None?

Reply:
I think Any brings less problems than None as a default.

Reply:
Right, so are we in agreement that it's better to use Any as a default, not not using None and alternating back and forth everywhere.

We can make these changes in a followup.

Reply:
Perfect.